boefjes/boefjes/plugins/kat_testssl_sh_ciphers/schema.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Arguments",
+  "type": "object",
+  "properties": {
+    "TIMEOUT": {
+      "title": "TIMEOUT",
+      "type": "integer",
+      "description": "Timeout for requests in the test ssl script",
+      "default": 30
```

Comment:
You need to include a `minimum` (of 0) property, otherwise this allows negative numbers